classes/filter_manager.php

```diff
@@ -150,4 +159,12 @@ public function invalidate_filters_cache() {
         $cache = cache::make('block_xp', 'filters');
         $cache->delete('filters_' . $this->manager->get_courseid());
     }
+
+    public static function save_default_filters() {
```

Comment:
Missing PHP Doc. Again, perhaps we better have a transparent handling of the filters using the courseid 0. I'm not sure, let's discuss it.